feature/session/src/main/res/layout/layout_chip.xml

```diff
@@ -4,6 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
+    android:layout_marginBottom="@dimen/session_filter_chip_vertical_margin"
```

Comment:
It was fixed in master. So probably this margin is not needed🙏

Reply:
sorry...and thanks.. I fixed